robocop/checkers/lengths.py

```diff
@@ -337,6 +338,41 @@
         version=">=6",
         added_in_version="3.3.0",
     ),
+    "0532": Rule(
+        RuleParam(
+            name="max_args",
+            default=1,
+            converter=int,
+            desc="maximum number of arguments allowed in the continuation line",
+        ),
+        rule_id="0532",
+        name="arguments-per-line",
+        msg="There is too many arguments per continuation line ({{ arguments_count }} / {{ max_arguments_count }})",
+        severity=RuleSeverity.INFO,
+        added_in_version="5.0.0",
+        docs="""
+        If the keyword's ``[Arguments]`` are split into multiple lines, it is recommended to put only one argument
+        per every line.
+        
+        Bad |:x:| ::
+
+            *** Keywords ***
+            Keyword With Multiple Arguments
+            [Arguments]    ${first_arg
+            ...    ${second_arg}    ${third_arg}=default
+
+         Good |:white_check_mark:| ::
+
+        ..  code-block:: none
```

Comment:
This looks wrong in the documentation output: 

![image](https://github.com/MarketSquare/robotframework-robocop/assets/50105763/88437cab-73f0-4c51-a5aa-d61f17697611)